### PR TITLE
added a basic check that will congratulate users on their first day o…

### PR DESCRIPTION
**Pull request by king0120**
…f #5 

Additionally set up a new Twit stream that checks for new tweets using "#100DaysOfCode"

### CODE REVIEW

**Comment by king0120** [comment]
Added a function to check for the last days

File: bot.js [hunk 1]
```diff
@@ -108,18 +109,43 @@ function tweetNow(tweetTxt) {
...
+  console.log(`Checking if first day`)
+  for (let i = 0; i < firstDay.length; i++) {    
+    if (checkTweetForText(tweet.text, firstDay[i])) {
+      return true;
+    }
+  }
+}
+
+function checkTweetForText(tweetText, value){
+  return tweetText.toLowerCase().indexOf(value) > -1 && text.toLowerCase().indexOf('100daysofcode') > -1
```
Comment (Ilyes-Hammadi) [inline bot.js]: `text.toLowerCase().indexOf('100daysofcode')` declaration !!

Comment (king0120) [inline bot.js]: whoops! You're right, I missed that when cleaning up the code.

**Comment by amandeepmittal** [comment]
@king0120 thank you, let me take a look at it. I hope nothing breaks. 
One suggestion, for future, please do not change the name of variables that already exists since it will break the code coming in other PRs.

**Comment by amandeepmittal** [comment]
All good, but there's one issue. When the bot is running, can you change the way it checks for the number `1` and `100` because if someone tweets `Day 1` or if someone tweets `Day 100` only `checkIfFirstDay` function runs which consists `Day 1`, however for `Day 100` the `checkIfLastDay` function should run.
See these:
When a user tweets Day 100
Bot could not recognise the Day100

For the time being I am merging your request, but I am looking forward to a way out.

**Pull request merged**